### ISSUE TITLE
Remote DB Connection

### ISSUE THREAD

**Issue by spitefultowel** [pwm-project/pwm#245]
I've setup PWM and am attempting to enable remote db. I'm using the latest snapshot and get the following error when testing connection. It's on a MariaDB 10.2 install.

> unable to initialize database: exception initializing database service: 5051 ERROR_DB_UNAVAILABLE (error creating new table PWM_META: (conn=22) You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'value TEXT )

**Comment by jrivard** [comment]
please ask for help in the pwm-general google group.

**Comment by spitefultowel** [comment]
Reopening as the issue isn't specific to the configuration of my PWM install. It appears that something happened with the SQL for MySQL based databases as of build 30195566 that causes it to not work. I've got an earlier version with the same config (28700112) that works as intended. I'll be reinstalling the earlier build to validate that it is something that has changed since I'm able to successfully connect to the database. I've already adjusted lower_case_table_names to both 0 and 1 which result in different errors unrelated to the actual connection to the database, but the execution of the SQL to create the tables in said database.
